content/tutorials/stream-metrics-from-telegraf-to-grafana.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Stream data from Telegraf to Grafana
+summary: Use Telegraf to stream live data into Grafana.
+id: stream-data-from-telegraf-to-grafana
+categories: ["administration"]
+tags: ["beginner"]
+status: Published
+authors: ["grafana_labs"]
+Feedback Link: https://github.com/grafana/tutorials/issues/new
+weight: 75
+---
+
+{{< tutorials/step title="Introduction" >}}
+
+Grafana v8 introduced streaming capabilities – a way to push data to UI panels in near real-time. In this tutorial we show how Grafana real-time streaming capabilities can be used together with Telegraf to instantly display system measurements.
+
+In this tutorial, you'll:
+
+- Setup Telegraf and output measurements directly to Grafana time-series panel in near real-time
+
+{{% class "prerequisite-section" %}}
+#### Prerequisites
+
+- Grafana 8.0+
+- Telegraf
+{{% /class %}}
+
+{{< /tutorials/step >}}
+{{< tutorials/step title="Run Grafana and create admin token" >}}
+
+1. Run Grafana following [installation instructions](https://grafana.com/docs/grafana/latest/installation/) for your operating system
+1. Log in and go to Configuration -> API Keys
+1. Press "Add API key" button and create a new API token with **Admin** role
+
+{{< /tutorials/step >}}
+{{< tutorials/step title="Configure and run Telegraf" >}}
+
+Telegraf is a plugin-driven server agent for collecting and sending metrics and events from databases, systems, and IoT sensors.
+
+You can install it following [official installation instructions](https://docs.influxdata.com/telegraf/v1.18/introduction/installation/).
+
+In this tutorial we will be using Telegraf HTTP output plugin to send metrics in Influx format to Grafana. We can use a configuration like this:
+
+```
+[agent]
+  interval = "1s"
+  flush_interval = "1s"
+
+[[inputs.cpu]]
+  percpu = false
+  totalcpu = true
+
+[[outputs.http]]
+  url = "http://localhost:3000/api/live/push/telegraf"
```

Comment:
```suggestion
  url = "http://localhost:3000/api/live/push/your-stream-key"
```
and update below.  Lets not make "telegraf" feel like it is a special key

Reply:
Done, replaced to `custom_stream_id` and mentioned that it can be changed